imagetest/test_suites/image_boot/image_boot_test.go

```diff
@@ -24,6 +25,10 @@ func TestGuestBoot(t *testing.T) {
 }
 
 func TestGuestReboot(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("Reboot not implemented on Windows")
```

Comment:
why is reboot not implemented on windows?

Reply:
oh ok for tests which don't work on a platform yet, no need to add these permanent skips. better just in your testing to specify what tests you *do* want to run with combination of `-filter` and -`exclude`, wdyt?

Reply:
Those flags only work on the whole package name right? I don't see a way to do that a per-test level within a package

Reply:
i see, i thought you were doing this for all tests in the 'suite'. what are the blockers to finishing the windows tests in this suite before merging? i'd rather not check in short-lived temporary skips if we can avoid it

Reply:
Actually I was able to do the whole suite.